src/content/api/module-methods.md

```diff
@@ -353,17 +353,28 @@ Aside from the module syntaxes described above, webpack also allows a few custom
 require.context(
   directory: String,
   includeSubdirs: Boolean /* optional, default true */,
-  filter: RegExp /* optional */
+  filter: RegExp /* optional, default /^\.\/.*$/, any file */,
+  mode: String  /* optional, 'sync' | 'eager' | 'weak' | 'lazy' | 'lazy-once', default 'sync' */
 )
 ```
 
-Specify a whole group of dependencies using a path to the `directory`, an option to `includeSubdirs`, and a `filter` for more fine grained control of the modules included. These can then be easily resolved later on:
+Specify a whole group of dependencies using a path to the `directory`, an option to `includeSubdirs`, with a `filter` for more fine grained control of the modules included, and a `mode` to define the way how loading will work. Underlying modules can then be easily resolved later on:
```

Comment:
`and` -> `with` change doesnt sound good to me, maybe just leave it as:
a path .... , an option ..., a filter ... and a mode

Reply:
Updated